python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -425,6 +485,11 @@ def resolve_repository(self, graphene_info) -> "GrapheneRepository":
             graphene_info.context.instance, self._external_repository, self._repository_location
         )
 
+    def resolve_required_resources(self, _graphene_info) -> Sequence[GrapheneResourceRequirement]:
+        node_def_snap = self.get_node_definition_snap()
```

Comment:
One more possible solution - we could extend this resolver to fetch the resources and return them rather than just the `all_unique_keys`. It might be somewhat slow, but if Dagit is going to turn around and make a second graphQL query for the resources we may as well do it on the backend in one request?